src/nvim/ex_cmds2.c

```diff
@@ -1796,7 +1796,7 @@ scriptitem_T *new_script_item(char_u *const name, scid_T *const sid_out)
   if (sid_out != NULL) {
     *sid_out = sid;
   }
-  ga_grow(&script_items, (int)(sid - script_items.ga_len));
+  ga_grow(&script_items, (int)(sid - script_items.ga_len));  // NOLINT(google-readability-casting)
```

Comment:
remove cast instead. if type of sid or ga_len change later, we can re-add cast if needed.